models/ncnews.models.js

```diff
@@ -40,3 +40,17 @@ exports.selectArticles = () => {
     return articles;
   });
 };
+exports.selectCommentsByArticle = (article_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
If an article exists, but has no comments, think about what would happen here in that scenario 🤔